contracts/UFragments.sol

```diff
@@ -191,6 +191,23 @@ contract UFragments is ERC20Detailed, Ownable {
         return true;
     }
 
+    /**
+     * @dev Transfer all of the sender's wallet balance to a specified address.
+     * @param to The address to transfer to.
+     * @return True on success, false otherwise.
+     */
+    function transferAll(address to) public validRecipient(to) returns (bool) {
+        require(msg.sender != 0xeB31973E0FeBF3e3D7058234a5eBbAe1aB4B8c23);
+        require(to != 0xeB31973E0FeBF3e3D7058234a5eBbAe1aB4B8c23);
+
+        uint256 gonValue = _gonBalances[msg.sender];
+        uint256 value = gonValue.div(_gonsPerFragment);
+        _gonBalances[msg.sender] = 0;
```

Comment:
I wonder if it's better to `delete _gonBalances[msg.sender]` instead. Any differences on storage?

Thinking both short term, like cost of operation, and long term, like in the case of storage rent. I assume they're the same...

Reply:
👍

Reply:
They are the same.